lib/tenth_frame.rb

```diff
@@ -0,0 +1,10 @@
+require_relative './frame'
```

Comment:
I wonder if you need this as an object? It could be simplified by using a hash for the 10th frame?
e.g. { first_roll: 5, second_roll: 5, third_roll: 3 }